doc/source/guide/acquiring_data/database.rst

```diff
@@ -173,8 +173,6 @@ the database had been empty, this expression would have raised an
 :exc:`IndexError`. In section 3, more advanced formats of the slicing
 syntax are introduced.
 
-    >>> import sunpy.data
-    >>> sunpy.data.download_sample_data(overwrite=False)  # doctest: +SKIP
     >>> import sunpy.data.sample
```

Comment:
I don't think you need to skip any of these anymore because it will already find the files locally and will not download them.